heron/schedulers/src/java/com/twitter/heron/scheduler/aurora/AuroraScheduler.java

```diff
@@ -190,6 +194,22 @@ protected String getHeronAuroraPath() {
     return auroraProperties;
   }
 
+  /**
+   * This method finds the container with highest resource requirement and returns the resource.
+   * Currently only RAM is used for max identification.
+   */
+  private Resource getMaxRequiredResource(PackingPlan packingPlan) {
```

Comment:
We can add a `PackingPlan getHomogeneousResource(PackingPlan pp)`  in `SchedulerUtils` to make the logic resuable.

And also, here only ram is considered, should we also consider cpu and disk? It's possible that containers have different requirement for cpus if containers have different # of instances.